api.yaml

```diff
@@ -159,7 +159,32 @@ tags:
         Example limits:
         * Adding 8 adults tickets leaves room for 2 child tickets.
 paths:
-  '/v1/products/{product_id}/dates':
+  "/v1/products":
+    get:
+      tags:
+        - "Product catalog"
+      summary: A catalog of available products.
+      description: Returns a list of available products.
+      operationId: getProducts
+      parameters:
+        - name: use_timeslots
```

Comment:
I would skip that. It should be as simple to implement as possible.

Reply:
I don't think one query param is a big implementation problem